### ISSUE TITLE
Doesn't work with web components with style attributes

### ISSUE THREAD

**Issue by sinchang** [remarkablemark/html-react-parser#188]
## Expected Behavior
render web component correctly

## Actual Behavior
throw error on build

## Reproducible Demo
https://codesandbox.io/s/html-react-parser-forked-vll1t

## Environment

- Version: [v0.14.2](https://github.com/remarkablemark/html-react-parser/tree/v0.14.2)

**Comment by remarkablemark** [comment]
I can confirm that this error for web components occurs even in React v15:

```
The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
```

I will work on a fix to ensure that [`attributesToProps`](https://github.com/remarkablemark/html-react-parser/blob/v0.14.2/lib/dom-to-react.js#L64-L66) will transform `style` string into an object for custom elements.

**Comment by remarkablemark** [comment]
@sinchang Please review PR #193

**Comment by remarkablemark** [comment]
Published [0.14.3](https://github.com/remarkablemark/html-react-parser/releases/tag/v0.14.3):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm i html-react-parser@0.14.3
```

[Yarn](https://yarnpkg.com/package/html-react-parser):

```sh
yarn add html-react-parser@0.14.3
```
